neuralforecast/models/timellm.py

```diff
@@ -256,14 +266,12 @@ class TimeLLM(BaseWindows):
     `stride`: int=8, stride of patch.<br>
     `d_ff`: int=128, dimension of fcn.<br>
     `top_k`: int=5, top tokens to consider.<br>
-    `d_llm`: int=768, hidden dimension of LLM.<br>
+    `d_llm`: int=768, hidden dimension of LLM.<br> # LLama7b:4096; GPT2-small:768; BERT-base:768
     `d_model`: int=32, dimension of model.<br>
     `n_heads`: int=8, number of heads in attention layer.<br>
     `enc_in`: int=7, encoder input size.<br>
     `dec_in`: int=7, decoder input size.<br>
-    `llm` = None, LLM model to use. If not specified, it will use GPT-2 from https://huggingface.co/openai-community/gpt2"<br>
-    `llm_config` = None, configuration of LLM. If not specified, it will use the configuration of GPT-2 from https://huggingface.co/openai-community/gpt2"<br>
-    `llm_tokenizer` = None, tokenizer of LLM. If not specified, it will use the GPT-2 tokenizer from https://huggingface.co/openai-community/gpt2"<br>
+    `llm_model` = GPT2, LLM model to use. If not specified, it will use GPT-2 from https://huggingface.co/openai-community/gpt2"<br> # LLAMA, GPT2, BERT
```

Comment:
Isn't it possible to re-use the existing variable names to prevent a breaking change? I'm fine with deprecating the llm_config and llm_tokenizer argument, but then we keep them and show users a deprecation message when they provide a non-default value

Reply:
I think saving those variables is what takes a long time and what this PR is for, I'd just suggest keeping the `llm` argument instead of also renaming it to reduce the scope of the breaking change.

Reply:
I’ve updated it to reuse the existing variable names as you suggested.